### PR TITLE
Fixed link to coffeescript

### PR DESCRIPTION
**Pull request by mishari**

### CODE REVIEW

**Comment by BIGjuevos** [comment]
CI build due to S3 being silly.

I'm giving this a :+1: and pinging @istr and @mickaelandrieu for their input

**Comment by mickaelandrieu** [comment]
I'd like to see a note to inform user that coffee script is not supported with phantomjs 2+ and slimerjs :)

**Comment by istr** [comment]
> CI build due to S3 being silly.

@BIGjuevos Yes, there seems to be a throttling in place with no exemption for travis build servers.
I restarted the build.

File: docs/quickstart.rst [hunk 1]
```diff
@@ -129,7 +129,7 @@ Run it:
 CoffeeScript version
 --------------------
 
-You can also write Casper scripts using the `CoffeeScript syntax <http://jashkenas.github.com/coffee-script/>`_:
+You can also write Casper scripts using the `CoffeeScript syntax <http://coffeescript.org/>`_:
```
Comment (istr) [inline docs/quickstart.rst]: ...with specific engines supporting it (that is, PhantomJS before 2.0.0).

**Comment by istr** [comment]
> I'd like to see a note to inform user that coffee script is not supported with phantomjs 2+ and slimerjs :)

Yes. Let us keep in mind to drop mention of coffeescript completely for CasperJS 2.0 (that might drop PhantomJS < 2.0.0 support).

**Comment by istr** [comment]
@mishari You could make a lot of people happy if you added some comment about the engines supporting coffeescript at all, right at the beginning of the document. See #1470.

**Comment by BIGjuevos** [comment]
I've commented about coffeescript support with #1510.

I think we can merge just this in now as it is just updating the URL.

**Comment by mickaelandrieu** [comment]
Sure :) thank you @mishari

**Pull request merged**